tests/scaletest/lib/utils.go

```diff
@@ -281,3 +283,187 @@ func FetchOPERDownVirtualService(t *testing.T, AviClient *clients.AviClient) []V
 	}
 	return OperDownVS
 }
+
+func CleanResourceData(data string) string {
+	data = strings.Replace(data, "\\", "", -1)
+	data = strings.Replace(data, " ", "", -1)
+	return data
+}
+
+func CompareVirtualServiceResources(t *testing.T, eventLog models.EventLog) bool {
+	var new, old models.VirtualService
+	err := json.Unmarshal([]byte(CleanResourceData(*eventLog.EventDetails.ConfigUpdateDetails.NewResourceData)), &new)
+	if err != nil {
+		t.Logf("Error unmarshalling data into VS. Error : %v", err)
```

Comment:
I think we should return here.

Reply:
Thanks for pointing that out. Will change it to Fatalf